index.html

```diff
@@ -27,7 +27,7 @@
         <meta charset="UTF-8">
 
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.6.1/dat.gui.min.js"></script>
+        <script src="node_modules/dat.gui/build/dat.gui.min.js"></script>
```

Comment:
I would prefer to have webpack deal only with the framework and not touch anything linked to "client app" (which `index.html` really is). Actually to be a bit more precise, here is the way I see things:

- index.html is indeed an example, but one that targets developers (it's the entrypoint when you do `npm start`). It then makes sense to reference dev dependencies in it.
- We could add  a bunch of other use cases in `examples/` (or better `demos/`). Then one, some or all of these demos would be included in the github pages of this repo. These client apps would do their own stuff about whatever ui, external js they need (use CDN, have a js/ folder, whatever).

So what I propose for this PR: copy the old version of `index.html` in `demos/`, use this new one for the github page and keep the new version of `index.html` here, for developers.

@peppsac is working on a plan mode, so this would be another demos, and I plan on adding others with wmts from other sources (from USA National Map for example). 

WDYT?